tests/lib/thrift/ThriftTest/src/server.hpp

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright 2022 Young Mei
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <cstdint>
+#include <cstdio>
+
+#include <sstream>
+#include <stdexcept>
+
+#include "SecondService.h"
+#include "ThriftTest.h"
+
+using namespace std;
+
+using namespace apache::thrift;
+using namespace apache::thrift::transport;
+using namespace thrift::test;
+
+class TestHandler : public ThriftTestIf {
+public:
+  TestHandler() = default;
+
+  void testVoid() override { printf("testVoid()\n"); }
+
+  void testString(string& out, const string& thing) override {
+    printf("testString(\"%s\")\n", thing.c_str());
+    out = thing;
+  }
+
+  bool testBool(const bool thing) override {
+    printf("testBool(%s)\n", thing ? "true" : "false");
+    return thing;
```

Comment:
Indentation issue in this file still

Reply:
I wish we had a clang-format that worked 😭

Reply:
Fixed!